.github/topissuebot.yml

```diff
@@ -0,0 +1,4 @@
+# Configuration for https://probot.github.io/apps/topissues/
+labelName: ":thumbsup: Top Issue!"

```

Comment:
Thinking of some better label for this (to fit a bit more in our current label scheme) `top-voted` ?